consensus/tendermint/action.go

```diff
@@ -0,0 +1,56 @@
+package tendermint
+
+import (
+	"time"
+)
+
+type Action[V Hashable[H], H Hash, A Addr] interface {
+	Execute(d *Driver[V, H, A])
+}
+
+type BroadcastProposal[V Hashable[H], H Hash, A Addr] Proposal[V, H, A]
+
+type BroadcastPrevote[V Hashable[H], H Hash, A Addr] Prevote[H, A]
+
+type BroadcastPrecommit[V Hashable[H], H Hash, A Addr] Precommit[H, A]
```

Comment:
Why do we define generic types `V` for these type which don't use it explicitly? Is it necessary or do you think it's good for legibility/extensibility?

Reply:
The main reason is that currently `Action` requires all of `V, H, A` to determint the type arguments of `Driver` in `Execute` function.

Reply:
Pushed the fix